universal-application-tool-0.0.1/app/views/applicant/ApplicantProgramSummaryView.java

```diff
@@ -39,6 +43,8 @@
 
   private final ApplicantLayout layout;
 
+  private static Logger LOG = LoggerFactory.getLogger(ApplicantProgramSummaryView.class);
```

Comment:
suggestion (blocking): All-caps is only for things that are final. Logger names should be lowercase. Rename to `logger`